src/installer/README.txt

```diff
@@ -0,0 +1,19 @@
+All files are available in '/usr/local/lib/odfe-sql-odbc' after installation.
+
+To setup a connection, you can use DSN to store your data source connection information,
+1. Open 'iODBC Data Source Administrator'.
+2. Go to 'User DSN'.
+3. Select 'ODFE SQL ODBC DSN' and click on 'Configure'.
+4. Update the connection string values. For the list of all supported options, check '/usr/local/lib/odfe-sql-odbc/doc/README.md'
+5. Click 'Ok' to save changes.
+
+If using with ODBC compatible BI tools,refer to the tool documentation on configuring a new ODBC driver. Typically, all that's required is to make the tool aware of the location of the driver library file and then use it to setup database (i.e Elasticsearch) connections.
```

Comment:
In general, for documentation, try to avoid contractions (ex: `that's -> that is`)
```suggestion
If using with ODBC compatible BI tools, refer to the tool documentation on configuring a new ODBC driver. Typically, all that is required is to make the tool aware of the location of the driver library file and then use it to setup database (i.e Elasticsearch) connections.
```

Reply:
done